go.mod

```diff
@@ -152,11 +152,11 @@ require (
 	golang.org/x/time v0.5.0 // indirect
 	gomodules.xyz/jsonpatch/v2 v2.4.0 // indirect
 	google.golang.org/appengine v1.6.8 // indirect
-	google.golang.org/protobuf v1.32.0 // indirect
+	google.golang.org/protobuf v1.33.0 // indirect
```

Comment:
Does the ci bot miss them?

Reply:
no, these deps requires manual updated look at the pr mentioned in the `Resolves` section